src/jobs/enrich-mustache-from-path-filter.yml

```
@@ -0,0 +1,79 @@
+description: >-
+  meant to be a drop in replacement for Circle's path-filtering/filter
+  BUT: meant to be used in a situation where EITHER you
+  want an easy way to use the rest of this orb's enhance-mustache
+  functionality OR find that Circle's official path-filtering orb
+  doesn't work for your use case anymore.
+  For exmaple, Circle's path-filtering works great if you want to turn on or off
+  currently defined workflows or part of a pipeline. But if you have
+  a situation where you want to create many instances of the SAME
+  jobs, this is much harder to keep DRY.
+
+  In enrich-mustache-from-path-filter instead of passing parameters
+  via CircleCI continutation pipeline, it renders the specified Circle
+  pipeline as a mustache document. This Circle job was created to make the
+  path from path-filter
+
+  Rules for parameters provided in the mappings section:
+    1. these must be JSON literals
+    2. If the JSON literal is an array [], if multiple matches are
+       found then all the matched values will be concatinated in an array
+    3. If a value is not marked as an array then subsequent sets will override
+
+docker:
+  - image: 'cimg/python:3.11'
+parameters:
+  base-revision:
+    description: Compare this branch to its ancestor named
+    type: string
+    default: main
+  config-path:
+    description: Location of the (mustache enriched) config file
+    type: string
+  mapping:
+    description: >-
+      Like path-filtering, the format is:
+        REGEX PARAMETER VALUE
+
+        (yes, space seperated values)
```

```suggestion
        (space seperated values)
```

Too conversational imo